Fix reaching EOL on Fedora 34 and 35 in repos

foxzi opened the pull request:
The support for Fedora 34 Linux ended on June 6, 2022 and the support for Fedora 35 Linux ended on December 13, 2022. The package repository has been moved to http://archives.fedoraproject.org. Fix url of repos in /etc/yum.repos.d/*.

foxzi committed:
Fix reaching EOL on Fedora 34 in repos

The support for Fedora Linux ended on June 7, 2022.
The package repository has been moved to
http://archives.fedoraproject.org. Fix url of repos in
/etc/yum.repos.d/*.

ylobankov commented on fedora/34/Dockerfile:
```diff
@@ -5,6 +5,13 @@ MAINTAINER Vitaliia Ioffe <v.ioffe@tarantool.org>
+# http://archives.fedoraproject.org.
+RUN sed -i 's/metalink=/#metalink=/g' /etc/yum.repos.d/*
+RUN sed -i 's/#baseurl=/baseurl=/g' /etc/yum.repos.d/*
+RUN sed -i 's/download.example/archives.fedoraproject.org/g' /etc/yum.repos.d/*
```
`sed -i 's/download.example\/pub/archives.fedoraproject.org\/pub\/archive/g' /etc/yum.repos.d/*`

foxzi replied:
done

foxzi committed:
Fix reaching EOL on Fedora 35 in repos

The support for Fedora Linux ended on December 13, 2022.
The package repository has been moved to
http://archives.fedoraproject.org. Fix url of repos in
/etc/yum.repos.d/*.

ylobankov commented on fedora/35/Dockerfile:
```diff
@@ -5,6 +5,14 @@ MAINTAINER Vitaliia Ioffe <v.ioffe@tarantool.org>
+RUN sed -i 's/#baseurl=/baseurl=/g' /etc/yum.repos.d/*
+RUN sed -i 's/download.example\/pub/archives.fedoraproject.org\/pub\/archive/g' /etc/yum.repos.d/*
+
+
```
Please delete this empty line.

foxzi replied:
done

ylobankov commented on fedora/34/Dockerfile:
```diff
@@ -5,6 +5,13 @@ MAINTAINER Vitaliia Ioffe <v.ioffe@tarantool.org>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# The support for Fedora Linux ended on June 6, 2022. 
```
`Jun 7, 2022` according to https://docs.fedoraproject.org/en-US/releases/eol/.

foxzi replied:
fixed

ylobankov reviewed (approved):
LGTM

Totktonada commented:
Looked on Fedora 34.

<details>
<summary>Before</summary>

```
/etc/yum.repos.d/fedora-cisco-openh264.repo:[fedora-cisco-openh264]
/etc/yum.repos.d/fedora-cisco-openh264.repo:name=Fedora $releasever openh264 (From Cisco) - $basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:type=rpm
/etc/yum.repos.d/fedora-cisco-openh264.repo:enabled=1
/etc/yum.repos.d/fedora-cisco-openh264.repo:metadata_expire=14d
/etc/yum.repos.d/fedora-cisco-openh264.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgcheck=1
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:skip_if_unavailable=True
/etc/yum.repos.d/fedora-cisco-openh264.repo:
/etc/yum.repos.d/fedora-cisco-openh264.repo:[fedora-cisco-openh264-debuginfo]
/etc/yum.repos.d/fedora-cisco-openh264.repo:name=Fedora $releasever openh264 (From Cisco) - $basearch - Debug
/etc/yum.repos.d/fedora-cisco-openh264.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-debug-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:type=rpm
/etc/yum.repos.d/fedora-cisco-openh264.repo:enabled=0
/etc/yum.repos.d/fedora-cisco-openh264.repo:metadata_expire=14d
/etc/yum.repos.d/fedora-cisco-openh264.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgcheck=1
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:skip_if_unavailable=True
/etc/yum.repos.d/fedora-cisco-openh264.repo:
/etc/yum.repos.d/fedora-modular.repo:[fedora-modular]
/etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - $basearch
/etc/yum.repos.d/fedora-modular.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Modular/$basearch/os/
/etc/yum.repos.d/fedora-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-modular.repo:enabled=1
/etc/yum.repos.d/fedora-modular.repo:countme=1
/etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
/etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-modular.repo:type=rpm
/etc/yum.repos.d/fedora-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-modular.repo:
/etc/yum.repos.d/fedora-modular.repo:[fedora-modular-debuginfo]
/etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - $basearch - Debug
/etc/yum.repos.d/fedora-modular.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Modular/$basearch/debug/tree/
/etc/yum.repos.d/fedora-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-debug-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-modular.repo:enabled=0
/etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
/etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-modular.repo:type=rpm
/etc/yum.repos.d/fedora-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-modular.repo:
/etc/yum.repos.d/fedora-modular.repo:[fedora-modular-source]
/etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - Source
/etc/yum.repos.d/fedora-modular.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Modular/source/tree/
/etc/yum.repos.d/fedora-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-source-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-modular.repo:enabled=0
/etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
/etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-modular.repo:type=rpm
/etc/yum.repos.d/fedora-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular]
/etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - $basearch - Updates
/etc/yum.repos.d/fedora-updates-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/$basearch/
/etc/yum.repos.d/fedora-updates-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:enabled=1
/etc/yum.repos.d/fedora-updates-modular.repo:countme=1
/etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-modular.repo:
/etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular-debuginfo]
/etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - $basearch - Updates - Debug
/etc/yum.repos.d/fedora-updates-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/$basearch/debug/
/etc/yum.repos.d/fedora-updates-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-modular.repo:
/etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular-source]
/etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - Updates Source
/etc/yum.repos.d/fedora-updates-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/SRPMS/
/etc/yum.repos.d/fedora-updates-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular]
/etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - $basearch - Test Updates
/etc/yum.repos.d/fedora-updates-testing-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Modular/$basearch/
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:countme=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing-modular.repo:
/etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular-debuginfo]
/etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - $basearch - Test Updates Debug
/etc/yum.repos.d/fedora-updates-testing-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/$basearch/debug/
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing-modular.repo:
/etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular-source]
/etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - Test Updates Source
/etc/yum.repos.d/fedora-updates-testing-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/SRPMS/
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing]
/etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - $basearch - Test Updates
/etc/yum.repos.d/fedora-updates-testing.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Everything/$basearch/
/etc/yum.repos.d/fedora-updates-testing.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing.repo:countme=1
/etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing.repo:
/etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing-debuginfo]
/etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - $basearch - Test Updates Debug
/etc/yum.repos.d/fedora-updates-testing.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Everything/$basearch/debug/
/etc/yum.repos.d/fedora-updates-testing.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing.repo:
/etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing-source]
/etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - Test Updates Source
/etc/yum.repos.d/fedora-updates-testing.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Everything/SRPMS/
/etc/yum.repos.d/fedora-updates-testing.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates.repo:[updates]
/etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - $basearch - Updates
/etc/yum.repos.d/fedora-updates.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/$basearch/
/etc/yum.repos.d/fedora-updates.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates.repo:enabled=1
/etc/yum.repos.d/fedora-updates.repo:countme=1
/etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates.repo:type=rpm
/etc/yum.repos.d/fedora-updates.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates.repo:
/etc/yum.repos.d/fedora-updates.repo:[updates-debuginfo]
/etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - $basearch - Updates - Debug
/etc/yum.repos.d/fedora-updates.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/$basearch/debug/
/etc/yum.repos.d/fedora-updates.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates.repo:enabled=0
/etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates.repo:type=rpm
/etc/yum.repos.d/fedora-updates.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates.repo:
/etc/yum.repos.d/fedora-updates.repo:[updates-source]
/etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - Updates Source
/etc/yum.repos.d/fedora-updates.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/SRPMS/
/etc/yum.repos.d/fedora-updates.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates.repo:enabled=0
/etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates.repo:type=rpm
/etc/yum.repos.d/fedora-updates.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora.repo:[fedora]
/etc/yum.repos.d/fedora.repo:name=Fedora $releasever - $basearch
/etc/yum.repos.d/fedora.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Everything/$basearch/os/
/etc/yum.repos.d/fedora.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-$releasever&arch=$basearch
/etc/yum.repos.d/fedora.repo:enabled=1
/etc/yum.repos.d/fedora.repo:countme=1
/etc/yum.repos.d/fedora.repo:metadata_expire=7d
/etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora.repo:type=rpm
/etc/yum.repos.d/fedora.repo:gpgcheck=1
/etc/yum.repos.d/fedora.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora.repo:
/etc/yum.repos.d/fedora.repo:[fedora-debuginfo]
/etc/yum.repos.d/fedora.repo:name=Fedora $releasever - $basearch - Debug
/etc/yum.repos.d/fedora.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Everything/$basearch/debug/tree/
/etc/yum.repos.d/fedora.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-debug-$releasever&arch=$basearch
/etc/yum.repos.d/fedora.repo:enabled=0
/etc/yum.repos.d/fedora.repo:metadata_expire=7d
/etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora.repo:type=rpm
/etc/yum.repos.d/fedora.repo:gpgcheck=1
/etc/yum.repos.d/fedora.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora.repo:
/etc/yum.repos.d/fedora.repo:[fedora-source]
/etc/yum.repos.d/fedora.repo:name=Fedora $releasever - Source
/etc/yum.repos.d/fedora.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Everything/source/tree/
/etc/yum.repos.d/fedora.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-source-$releasever&arch=$basearch
/etc/yum.repos.d/fedora.repo:enabled=0
/etc/yum.repos.d/fedora.repo:metadata_expire=7d
/etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora.repo:type=rpm
/etc/yum.repos.d/fedora.repo:gpgcheck=1
/etc/yum.repos.d/fedora.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora.repo:skip_if_unavailable=False
```
</details>


<details>
<summary>Diff</summary>

```diff
--- /tmp/fedora-34-repos-orig.txt	2023-03-27 18:02:52.741977247 +0300
+++ /tmp/fedora-34-repos-new.txt	2023-03-27 18:25:04.918047264 +0300
@@ -1,6 +1,6 @@
 /etc/yum.repos.d/fedora-cisco-openh264.repo:[fedora-cisco-openh264]
 /etc/yum.repos.d/fedora-cisco-openh264.repo:name=Fedora $releasever openh264 (From Cisco) - $basearch
-/etc/yum.repos.d/fedora-cisco-openh264.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-cisco-openh264.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-cisco-openh264.repo:type=rpm
 /etc/yum.repos.d/fedora-cisco-openh264.repo:enabled=1
 /etc/yum.repos.d/fedora-cisco-openh264.repo:metadata_expire=14d
@@ -11,7 +11,7 @@
 /etc/yum.repos.d/fedora-cisco-openh264.repo:
 /etc/yum.repos.d/fedora-cisco-openh264.repo:[fedora-cisco-openh264-debuginfo]
 /etc/yum.repos.d/fedora-cisco-openh264.repo:name=Fedora $releasever openh264 (From Cisco) - $basearch - Debug
-/etc/yum.repos.d/fedora-cisco-openh264.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-debug-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-cisco-openh264.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-debug-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-cisco-openh264.repo:type=rpm
 /etc/yum.repos.d/fedora-cisco-openh264.repo:enabled=0
 /etc/yum.repos.d/fedora-cisco-openh264.repo:metadata_expire=14d
@@ -22,8 +22,8 @@
 /etc/yum.repos.d/fedora-cisco-openh264.repo:
 /etc/yum.repos.d/fedora-modular.repo:[fedora-modular]
 /etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - $basearch
-/etc/yum.repos.d/fedora-modular.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Modular/$basearch/os/
-/etc/yum.repos.d/fedora-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Modular/$basearch/os/
+/etc/yum.repos.d/fedora-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-modular.repo:enabled=1
 /etc/yum.repos.d/fedora-modular.repo:countme=1
 /etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
@@ -35,8 +35,8 @@
 /etc/yum.repos.d/fedora-modular.repo:
 /etc/yum.repos.d/fedora-modular.repo:[fedora-modular-debuginfo]
 /etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - $basearch - Debug
-/etc/yum.repos.d/fedora-modular.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Modular/$basearch/debug/tree/
-/etc/yum.repos.d/fedora-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-debug-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Modular/$basearch/debug/tree/
+/etc/yum.repos.d/fedora-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-debug-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
 /etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
@@ -47,8 +47,8 @@
 /etc/yum.repos.d/fedora-modular.repo:
 /etc/yum.repos.d/fedora-modular.repo:[fedora-modular-source]
 /etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - Source
-/etc/yum.repos.d/fedora-modular.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Modular/source/tree/
-/etc/yum.repos.d/fedora-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-source-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Modular/source/tree/
+/etc/yum.repos.d/fedora-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-source-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
 /etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
@@ -58,8 +58,8 @@
 /etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
 /etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular]
 /etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - $basearch - Updates
-/etc/yum.repos.d/fedora-updates-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/$basearch/
-/etc/yum.repos.d/fedora-updates-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/$basearch/
+/etc/yum.repos.d/fedora-updates-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-modular.repo:enabled=1
 /etc/yum.repos.d/fedora-updates-modular.repo:countme=1
 /etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
@@ -71,8 +71,8 @@
 /etc/yum.repos.d/fedora-updates-modular.repo:
 /etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular-debuginfo]
 /etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - $basearch - Updates - Debug
-/etc/yum.repos.d/fedora-updates-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/$basearch/debug/
-/etc/yum.repos.d/fedora-updates-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-debug-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/$basearch/debug/
+/etc/yum.repos.d/fedora-updates-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-debug-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
@@ -83,8 +83,8 @@
 /etc/yum.repos.d/fedora-updates-modular.repo:
 /etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular-source]
 /etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - Updates Source
-/etc/yum.repos.d/fedora-updates-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/SRPMS/
-/etc/yum.repos.d/fedora-updates-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-source-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/SRPMS/
+/etc/yum.repos.d/fedora-updates-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-source-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
@@ -94,8 +94,8 @@
 /etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular]
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - $basearch - Test Updates
-/etc/yum.repos.d/fedora-updates-testing-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Modular/$basearch/
-/etc/yum.repos.d/fedora-updates-testing-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-testing-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Modular/$basearch/
+/etc/yum.repos.d/fedora-updates-testing-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:countme=1
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
@@ -107,8 +107,8 @@
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular-debuginfo]
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - $basearch - Test Updates Debug
-/etc/yum.repos.d/fedora-updates-testing-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/$basearch/debug/
-/etc/yum.repos.d/fedora-updates-testing-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-debug-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-testing-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/$basearch/debug/
+/etc/yum.repos.d/fedora-updates-testing-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-debug-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
@@ -119,8 +119,8 @@
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular-source]
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - Test Updates Source
-/etc/yum.repos.d/fedora-updates-testing-modular.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Modular/SRPMS/
-/etc/yum.repos.d/fedora-updates-testing-modular.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-source-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-testing-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/SRPMS/
+/etc/yum.repos.d/fedora-updates-testing-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-source-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
@@ -130,8 +130,8 @@
 /etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
 /etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing]
 /etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - $basearch - Test Updates
-/etc/yum.repos.d/fedora-updates-testing.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Everything/$basearch/
-/etc/yum.repos.d/fedora-updates-testing.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-testing.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Everything/$basearch/
+/etc/yum.repos.d/fedora-updates-testing.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-testing.repo:countme=1
 /etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
@@ -143,8 +143,8 @@
 /etc/yum.repos.d/fedora-updates-testing.repo:
 /etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing-debuginfo]
 /etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - $basearch - Test Updates Debug
-/etc/yum.repos.d/fedora-updates-testing.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Everything/$basearch/debug/
-/etc/yum.repos.d/fedora-updates-testing.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-debug-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-testing.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Everything/$basearch/debug/
+/etc/yum.repos.d/fedora-updates-testing.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-debug-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
@@ -155,8 +155,8 @@
 /etc/yum.repos.d/fedora-updates-testing.repo:
 /etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing-source]
 /etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - Test Updates Source
-/etc/yum.repos.d/fedora-updates-testing.repo:#baseurl=http://download.example/pub/fedora/linux/updates/testing/$releasever/Everything/SRPMS/
-/etc/yum.repos.d/fedora-updates-testing.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-source-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates-testing.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Everything/SRPMS/
+/etc/yum.repos.d/fedora-updates-testing.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-source-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
 /etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
@@ -166,8 +166,8 @@
 /etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
 /etc/yum.repos.d/fedora-updates.repo:[updates]
 /etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - $basearch - Updates
-/etc/yum.repos.d/fedora-updates.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/$basearch/
-/etc/yum.repos.d/fedora-updates.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Everything/$basearch/
+/etc/yum.repos.d/fedora-updates.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates.repo:enabled=1
 /etc/yum.repos.d/fedora-updates.repo:countme=1
 /etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
@@ -179,8 +179,8 @@
 /etc/yum.repos.d/fedora-updates.repo:
 /etc/yum.repos.d/fedora-updates.repo:[updates-debuginfo]
 /etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - $basearch - Updates - Debug
-/etc/yum.repos.d/fedora-updates.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/$basearch/debug/
-/etc/yum.repos.d/fedora-updates.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-debug-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Everything/$basearch/debug/
+/etc/yum.repos.d/fedora-updates.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-debug-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates.repo:enabled=0
 /etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates.repo:type=rpm
@@ -191,8 +191,8 @@
 /etc/yum.repos.d/fedora-updates.repo:
 /etc/yum.repos.d/fedora-updates.repo:[updates-source]
 /etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - Updates Source
-/etc/yum.repos.d/fedora-updates.repo:#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/SRPMS/
-/etc/yum.repos.d/fedora-updates.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-source-f$releasever&arch=$basearch
+/etc/yum.repos.d/fedora-updates.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Everything/SRPMS/
+/etc/yum.repos.d/fedora-updates.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-source-f$releasever&arch=$basearch
 /etc/yum.repos.d/fedora-updates.repo:enabled=0
 /etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
 /etc/yum.repos.d/fedora-updates.repo:type=rpm
@@ -202,8 +202,8 @@
 /etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
 /etc/yum.repos.d/fedora.repo:[fedora]
 /etc/yum.repos.d/fedora.repo:name=Fedora $releasever - $basearch
-/etc/yum.repos.d/fedora.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Everything/$basearch/os/
-/etc/yum.repos.d/fedora.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/os/
+/etc/yum.repos.d/fedora.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora.repo:enabled=1
 /etc/yum.repos.d/fedora.repo:countme=1
 /etc/yum.repos.d/fedora.repo:metadata_expire=7d
@@ -215,8 +215,8 @@
 /etc/yum.repos.d/fedora.repo:
 /etc/yum.repos.d/fedora.repo:[fedora-debuginfo]
 /etc/yum.repos.d/fedora.repo:name=Fedora $releasever - $basearch - Debug
-/etc/yum.repos.d/fedora.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Everything/$basearch/debug/tree/
-/etc/yum.repos.d/fedora.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-debug-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/debug/tree/
+/etc/yum.repos.d/fedora.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-debug-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora.repo:enabled=0
 /etc/yum.repos.d/fedora.repo:metadata_expire=7d
 /etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
@@ -227,8 +227,8 @@
 /etc/yum.repos.d/fedora.repo:
 /etc/yum.repos.d/fedora.repo:[fedora-source]
 /etc/yum.repos.d/fedora.repo:name=Fedora $releasever - Source
-/etc/yum.repos.d/fedora.repo:#baseurl=http://download.example/pub/fedora/linux/releases/$releasever/Everything/source/tree/
-/etc/yum.repos.d/fedora.repo:metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-source-$releasever&arch=$basearch
+/etc/yum.repos.d/fedora.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/source/tree/
+/etc/yum.repos.d/fedora.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-source-$releasever&arch=$basearch
 /etc/yum.repos.d/fedora.repo:enabled=0
 /etc/yum.repos.d/fedora.repo:metadata_expire=7d
 /etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
```
</details>


<details>
<summary>After</summary>

```
/etc/yum.repos.d/fedora-cisco-openh264.repo:[fedora-cisco-openh264]
/etc/yum.repos.d/fedora-cisco-openh264.repo:name=Fedora $releasever openh264 (From Cisco) - $basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:type=rpm
/etc/yum.repos.d/fedora-cisco-openh264.repo:enabled=1
/etc/yum.repos.d/fedora-cisco-openh264.repo:metadata_expire=14d
/etc/yum.repos.d/fedora-cisco-openh264.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgcheck=1
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:skip_if_unavailable=True
/etc/yum.repos.d/fedora-cisco-openh264.repo:
/etc/yum.repos.d/fedora-cisco-openh264.repo:[fedora-cisco-openh264-debuginfo]
/etc/yum.repos.d/fedora-cisco-openh264.repo:name=Fedora $releasever openh264 (From Cisco) - $basearch - Debug
/etc/yum.repos.d/fedora-cisco-openh264.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-debug-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:type=rpm
/etc/yum.repos.d/fedora-cisco-openh264.repo:enabled=0
/etc/yum.repos.d/fedora-cisco-openh264.repo:metadata_expire=14d
/etc/yum.repos.d/fedora-cisco-openh264.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgcheck=1
/etc/yum.repos.d/fedora-cisco-openh264.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-cisco-openh264.repo:skip_if_unavailable=True
/etc/yum.repos.d/fedora-cisco-openh264.repo:
/etc/yum.repos.d/fedora-modular.repo:[fedora-modular]
/etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - $basearch
/etc/yum.repos.d/fedora-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Modular/$basearch/os/
/etc/yum.repos.d/fedora-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-modular.repo:enabled=1
/etc/yum.repos.d/fedora-modular.repo:countme=1
/etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
/etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-modular.repo:type=rpm
/etc/yum.repos.d/fedora-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-modular.repo:
/etc/yum.repos.d/fedora-modular.repo:[fedora-modular-debuginfo]
/etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - $basearch - Debug
/etc/yum.repos.d/fedora-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Modular/$basearch/debug/tree/
/etc/yum.repos.d/fedora-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-debug-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-modular.repo:enabled=0
/etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
/etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-modular.repo:type=rpm
/etc/yum.repos.d/fedora-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-modular.repo:
/etc/yum.repos.d/fedora-modular.repo:[fedora-modular-source]
/etc/yum.repos.d/fedora-modular.repo:name=Fedora Modular $releasever - Source
/etc/yum.repos.d/fedora-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Modular/source/tree/
/etc/yum.repos.d/fedora-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-source-$releasever&arch=$basearch
/etc/yum.repos.d/fedora-modular.repo:enabled=0
/etc/yum.repos.d/fedora-modular.repo:metadata_expire=7d
/etc/yum.repos.d/fedora-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-modular.repo:type=rpm
/etc/yum.repos.d/fedora-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular]
/etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - $basearch - Updates
/etc/yum.repos.d/fedora-updates-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/$basearch/
/etc/yum.repos.d/fedora-updates-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:enabled=1
/etc/yum.repos.d/fedora-updates-modular.repo:countme=1
/etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-modular.repo:
/etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular-debuginfo]
/etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - $basearch - Updates - Debug
/etc/yum.repos.d/fedora-updates-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/$basearch/debug/
/etc/yum.repos.d/fedora-updates-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-modular.repo:
/etc/yum.repos.d/fedora-updates-modular.repo:[updates-modular-source]
/etc/yum.repos.d/fedora-updates-modular.repo:name=Fedora Modular $releasever - Updates Source
/etc/yum.repos.d/fedora-updates-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/SRPMS/
/etc/yum.repos.d/fedora-updates-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular]
/etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - $basearch - Test Updates
/etc/yum.repos.d/fedora-updates-testing-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Modular/$basearch/
/etc/yum.repos.d/fedora-updates-testing-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:countme=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing-modular.repo:
/etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular-debuginfo]
/etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - $basearch - Test Updates Debug
/etc/yum.repos.d/fedora-updates-testing-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/$basearch/debug/
/etc/yum.repos.d/fedora-updates-testing-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing-modular.repo:
/etc/yum.repos.d/fedora-updates-testing-modular.repo:[updates-testing-modular-source]
/etc/yum.repos.d/fedora-updates-testing-modular.repo:name=Fedora Modular $releasever - Test Updates Source
/etc/yum.repos.d/fedora-updates-testing-modular.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Modular/SRPMS/
/etc/yum.repos.d/fedora-updates-testing-modular.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-modular-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing-modular.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing-modular.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing-modular.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing-modular.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing]
/etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - $basearch - Test Updates
/etc/yum.repos.d/fedora-updates-testing.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Everything/$basearch/
/etc/yum.repos.d/fedora-updates-testing.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing.repo:countme=1
/etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing.repo:
/etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing-debuginfo]
/etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - $basearch - Test Updates Debug
/etc/yum.repos.d/fedora-updates-testing.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Everything/$basearch/debug/
/etc/yum.repos.d/fedora-updates-testing.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates-testing.repo:
/etc/yum.repos.d/fedora-updates-testing.repo:[updates-testing-source]
/etc/yum.repos.d/fedora-updates-testing.repo:name=Fedora $releasever - Test Updates Source
/etc/yum.repos.d/fedora-updates-testing.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/Everything/SRPMS/
/etc/yum.repos.d/fedora-updates-testing.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:enabled=0
/etc/yum.repos.d/fedora-updates-testing.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates-testing.repo:type=rpm
/etc/yum.repos.d/fedora-updates-testing.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates-testing.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates-testing.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates-testing.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates.repo:[updates]
/etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - $basearch - Updates
/etc/yum.repos.d/fedora-updates.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Everything/$basearch/
/etc/yum.repos.d/fedora-updates.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates.repo:enabled=1
/etc/yum.repos.d/fedora-updates.repo:countme=1
/etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates.repo:type=rpm
/etc/yum.repos.d/fedora-updates.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates.repo:
/etc/yum.repos.d/fedora-updates.repo:[updates-debuginfo]
/etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - $basearch - Updates - Debug
/etc/yum.repos.d/fedora-updates.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Everything/$basearch/debug/
/etc/yum.repos.d/fedora-updates.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-debug-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates.repo:enabled=0
/etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates.repo:type=rpm
/etc/yum.repos.d/fedora-updates.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora-updates.repo:
/etc/yum.repos.d/fedora-updates.repo:[updates-source]
/etc/yum.repos.d/fedora-updates.repo:name=Fedora $releasever - Updates Source
/etc/yum.repos.d/fedora-updates.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/Everything/SRPMS/
/etc/yum.repos.d/fedora-updates.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-source-f$releasever&arch=$basearch
/etc/yum.repos.d/fedora-updates.repo:enabled=0
/etc/yum.repos.d/fedora-updates.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora-updates.repo:type=rpm
/etc/yum.repos.d/fedora-updates.repo:gpgcheck=1
/etc/yum.repos.d/fedora-updates.repo:metadata_expire=6h
/etc/yum.repos.d/fedora-updates.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora-updates.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora.repo:[fedora]
/etc/yum.repos.d/fedora.repo:name=Fedora $releasever - $basearch
/etc/yum.repos.d/fedora.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/os/
/etc/yum.repos.d/fedora.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-$releasever&arch=$basearch
/etc/yum.repos.d/fedora.repo:enabled=1
/etc/yum.repos.d/fedora.repo:countme=1
/etc/yum.repos.d/fedora.repo:metadata_expire=7d
/etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora.repo:type=rpm
/etc/yum.repos.d/fedora.repo:gpgcheck=1
/etc/yum.repos.d/fedora.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora.repo:
/etc/yum.repos.d/fedora.repo:[fedora-debuginfo]
/etc/yum.repos.d/fedora.repo:name=Fedora $releasever - $basearch - Debug
/etc/yum.repos.d/fedora.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/debug/tree/
/etc/yum.repos.d/fedora.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-debug-$releasever&arch=$basearch
/etc/yum.repos.d/fedora.repo:enabled=0
/etc/yum.repos.d/fedora.repo:metadata_expire=7d
/etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora.repo:type=rpm
/etc/yum.repos.d/fedora.repo:gpgcheck=1
/etc/yum.repos.d/fedora.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora.repo:skip_if_unavailable=False
/etc/yum.repos.d/fedora.repo:
/etc/yum.repos.d/fedora.repo:[fedora-source]
/etc/yum.repos.d/fedora.repo:name=Fedora $releasever - Source
/etc/yum.repos.d/fedora.repo:baseurl=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/source/tree/
/etc/yum.repos.d/fedora.repo:#metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-source-$releasever&arch=$basearch
/etc/yum.repos.d/fedora.repo:enabled=0
/etc/yum.repos.d/fedora.repo:metadata_expire=7d
/etc/yum.repos.d/fedora.repo:repo_gpgcheck=0
/etc/yum.repos.d/fedora.repo:type=rpm
/etc/yum.repos.d/fedora.repo:gpgcheck=1
/etc/yum.repos.d/fedora.repo:gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
/etc/yum.repos.d/fedora.repo:skip_if_unavailable=False
```
</details>

Totktonada commented:
There is no repository URL for the `fedora-cisco-openh264` repository (metalink URL is commented and there are no other ones). `dnf makecache` reports a warning:

```
Error: Cannot find a valid baseurl for repo: fedora-cisco-openh264
<...>
Ignoring repositories: fedora-cisco-openh264
```

But exists with zero exit code. We can disable the repository to eliminate the warning. It's ok to do separately.
